scripts/ci/changelog/bin/changelog

```diff
@@ -132,9 +132,9 @@ else
             polkadot: $polkadot[0],
           srtool: [
             { name: "shell", data: $srtool_shell[0] },
-            { name: "westmint", data: $srtool_westmint[0] },
-            { name: "statemine", data: $srtool_statemine[0] },
-            { name: "statemint", data: $srtool_statemint[0] },
+            { name: "asset hub westend", data: $srtool_westmint[0] },
+            { name: "asset hub kusama", data: $srtool_statemine[0] },
+            { name: "asset hub polkadot", data: $srtool_statemint[0] },
```

Comment:
If you got for the version with spaces, we can add the caps as well:
`Asset Hub Westend`...

Reply:
Due to this one `### {{ runtime.name | capitalize }}` in the template it will be re-formatted to Asset hub westend